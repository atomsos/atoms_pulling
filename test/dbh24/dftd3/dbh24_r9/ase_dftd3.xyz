3

C       0.006610730173479      0.012167714561826     -0.532911414207949
N       0.009619533239263      0.007872479552476      0.628987241257140
H       0.418301526136076      0.317122130467271      0.724136397832129
