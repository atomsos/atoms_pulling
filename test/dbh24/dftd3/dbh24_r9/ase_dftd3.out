  _________________________________
                                   
 |         DFTD3 V3.1 Rev 1        |
 | S.Grimme, University Bonn       |
 |         October  2015           |
 |   see dftd3 -h for options      |
  _________________________________

 Please cite DFT-D3 work done with this code as:
 S. Grimme, J. Antony, S. Ehrlich and H. Krieg,
 J. Chem. Phys. 132 (2010), 154104
 If used with BJ-damping cite also
 S. Grimme, S. Ehrlich and L. Goerigk,
 J. Comput. Chem. 32 (2011), 1456-1465
 For DFT-D2 the reference is
 S. Grimme, J. Comput. Chem., 27 (2006), 1787-1799
 For DFT-D3M or DFT-D3M(BJ) the reference is
 D.G.A. Smith, L.A. Burns, K. Patkowski, and 
 C.D. Sherrill, J. Phys. Chem. Lett. 7 (2016) 2197-2203

  files read :     
 ase_dftd3.xyz
C6 coefficients used:
           2  C6 for element            1
Z=  1 CN= 0.912     C6(AA)=    3.03
Z=  1 CN= 0.000     C6(AA)=    7.59
           5  C6 for element            6
Z=  6 CN= 0.000     C6(AA)=   49.11
Z=  6 CN= 0.987     C6(AA)=   43.25
Z=  6 CN= 1.998     C6(AA)=   29.36
Z=  6 CN= 2.999     C6(AA)=   25.78
Z=  6 CN= 3.984     C6(AA)=   18.21
           4  C6 for element            7
Z=  7 CN= 0.000     C6(AA)=   25.27
Z=  7 CN= 0.994     C6(AA)=   22.12
Z=  7 CN= 2.014     C6(AA)=   19.68
Z=  7 CN= 2.990     C6(AA)=   15.58

#               XYZ [au]               R0(AA) [Ang.]  CN       C6(AA)     C8(AA)   C10(AA) [au] 
   1   0.01249   0.02299  -1.00706   c     1.455    1.693        31.4       908.8     32198.6
   2   0.01818   0.01488   1.18861   n     1.311    2.000        19.6       433.1     11704.9
   3   0.79048   0.59927   1.36842   h     1.091    1.693         3.0        36.6       541.9

molecular C6(AA) [au] =       138.19
 --------------------------------------------------
 !! SOME DISTANCES VERY SHORT. CHECK COORDINATES !!
 --------------------------------------------------

           DFT-D V3
 DF pbe                                               
 parameters
 s6       :    1.0000
 s8       :    0.7220
 rs6      :    1.2170
 rs18     :    1.0000
 alpha6   :   14.0000
 alpha8   :   16.0000
 k1-k3    :   16.0000    1.3333   -4.0000
 Cutoff   :   95.0000 a.u.
 CN-Cutoff:   40.0000 a.u.

 Edisp /kcal,au:    -0.0008 -0.00000122

 E6    /kcal :    -0.0001
 E8    /kcal :    -0.0007
 % E8        : 90.92

 doing analytical gradient O(N^2) ...

 |G|=   1.1861173086698996E-005
gdisp time     0.0
 Cartesian gradient written to file dftd3_gradient
 normal termination of dftd3
